sympy/calculus/tests/test_finite_diff.py

```diff
@@ -142,8 +142,8 @@ def test_differentiate_finite():
     res2 = differentiate_finite(f(x) + x**3 + 42, x, points=[x-1, x+1])
     ref2 = (f(x + 1) + (x + 1)**3 - f(x - 1) - (x - 1)**3)/2
     assert (res2 - ref2).simplify() == 0
-    raises(ValueError, lambda: differentiate_finite(f(x)*g(x), x,
-                                                    pints=[x-1, x+1]))
+    raises(TypeError, lambda: differentiate_finite(f(x)*g(x), x,
+                                                   pints=[x-1, x+1]))
```

Comment:
Should this be points rather than pints?

Reply:
The whole point in this test is to check a typo in an argument name, so this is correct as is

Reply:
It looks like it was added to increase coverage i.e. to cover the `raise ValueError` (that you've now deleted).

Reply:
Do you want me to remove it? Leaving it in acts as a record that the error type changed.

Reply:
I don't mind really. I just looked up why it was added because it doesn't seem like a very useful test